guide/designing-products/common-user-flows.md

```diff
@@ -450,7 +464,16 @@ This is not only helpful to users but can also help improve privacy. Since the i
    layout = "float-left"
 %}
 
-In some circumstances, users need to move all their funds to a new wallet. This may be because a wallet is potentially compromised, a simple change of software, or a migration to a wallet with a security setup that is more appropriate to the number of funds stored. Whichever it may be, this can be a vital function for users that applications should support. While it is easy to send all funds to a new address simply, additional meta and state data stored in wallet applications also need to be considered.
+There are several reasons a user might want to switch wallets. 
+
+A different wallet application might have features they need, or be better supported than the one they originally created the Bitcoin wallet with. Importing the wallet with the recovery phrase into the new application should be possible, and will be free from fees as no transfer of funds is happening.
+
+The owner may want to increase the security of their wallet, either by using a single-key wallet with a [passphrase](/guide/glossary/#passphrase), or a [multi-key](/guide/private-key-management/multi-key/) wallet. As both of these include transferring funds to a new Bitcoin wallet, there will be fees to pay.
+
+In the worst case scenario the wallet might have been compromised, and funds should be saved by sending them all to a different Bitcoin wallet.
+
+Whatever the reason may be, the import and backup of wallets is a vital function for users that applications should support. While it is easy to send all funds to a new address, additional meta and state data stored in wallet applications also need to be considered for full compatibility. Extra care needs to be taken when switching wallets that include funds on the Lightning network, as standards for backing up channel state is yet to emerge. See also, [Wallet interoperability](/guide/designing-products/wallet-interoperability/).
```

Comment:
Yes, I think that would be a great recommendation to make.

I agree it's theoretically possible, but I think it should be considered "for advanced users only" until the process becomes more standardized.

I opened a [discussion point about this on Slack](https://bitcoindesign.slack.com/archives/C01TTD8Q7ST/p1633534098104800) in case anybody has counter-examples to this.